credentialz/README.md

```diff
@@ -341,6 +341,45 @@ stream.Send(
 )
 ```
 
+### Rotate Certificate based on existing key.
+
+The most common operation we are expecting to require on devices is the rotation of certificates used for SSH access for devices. This operation expects to reuse the existing host key on the device as there is not really a good reason frequently rotate this identity.
```

Comment:
s/frequently rotate/to frequently rotate/ would be more grammatically correct.

Ending the sentence at "device", as in "This operation expects to reuse the existing host key on the device." would be better advice for general users, imho.